### ISSUE TITLE
Crash: LocationGMS.java line 203 com.onesignal.LocationGMS.getLocation

### ISSUE THREAD

**Issue by andpanda** [OneSignal/OneSignal-Android-SDK#1101]
After I have updated OneSignal unity sdk to the latest version (2.13.0) we got some crashes on crashlytics.

This issue has 25 crashes affecting 23 users
Devices: 
Galaxy S10
16%
Galaxy A30
8%
Galaxy Tab A
4%
Galaxy Note8
4%
Galaxy S10+
4%
Galaxy S7 edge
4%
Galaxy C5
4%
Galaxy A6+
4%
Galaxy A5(2017)
4%
Galaxy A20s
4%
Galaxy A10

Android Version:
60%
Android 10
16%
Android 8
16%
Android 9
8%
Android 7

Unity Version 2018.4.24 LTS
OneSignal 2.13.0

```
Fatal Exception: java.lang.Error: FATAL EXCEPTION [main]
Unity version     : 2018.4.24f1
Device model      : HMD Global Nokia C1
Device fingerprint: Nokia/Rocket_00WW/RKU:9/PPR1.180610.011/00WW_1_050:user/release-keys

Caused by java.lang.RuntimeException: Unable to resume activity {com.yallaplay.tarneebmasters/com.unity3d.player.UnityPlayerActivity}: java.lang.RuntimeException: android.os.DeadSystemException
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3837)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3869)
       at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6852)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:504)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

Caused by java.lang.RuntimeException: android.os.DeadSystemException
       at android.app.ApplicationPackageManager.getPackageInfoAsUser(ApplicationPackageManager.java:182)
       at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:155)
       at com.onesignal.LocationGMS.getLocation(LocationGMS.java:203)
       at com.onesignal.OneSignal.startLocationUpdate(OneSignal.java:964)
       at com.onesignal.OneSignal.startRegistrationOrOnSession(OneSignal.java:941)
       at com.onesignal.OneSignal.doSessionInit(OneSignal.java:846)
       at com.onesignal.OneSignal.onAppFocus(OneSignal.java:1279)
       at com.onesignal.ActivityLifecycleHandler.handleFocus(ActivityLifecycleHandler.java:213)
       at com.onesignal.ActivityLifecycleHandler.onActivityResumed(ActivityLifecycleHandler.java:127)
       at com.onesignal.ActivityLifecycleListener.onActivityResumed(ActivityLifecycleListener.java:78)
       at android.app.Application.dispatchActivityResumed(Application.java:239)
       at android.app.Activity.onResume(Activity.java:1362)
       at com.unity3d.player.UnityPlayerActivity.onResume()
       at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1434)
       at android.app.Activity.performResume(Activity.java:8762)
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3829)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3869)
       at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6852)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:504)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

Caused by android.os.DeadSystemException
       at android.app.ApplicationPackageManager.getPackageInfoAsUser(ApplicationPackageManager.java:182)
       at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:155)
       at com.onesignal.LocationGMS.getLocation(LocationGMS.java:203)
       at com.onesignal.OneSignal.startLocationUpdate(OneSignal.java:964)
       at com.onesignal.OneSignal.startRegistrationOrOnSession(OneSignal.java:941)
       at com.onesignal.OneSignal.doSessionInit(OneSignal.java:846)
       at com.onesignal.OneSignal.onAppFocus(OneSignal.java:1279)
       at com.onesignal.ActivityLifecycleHandler.handleFocus(ActivityLifecycleHandler.java:213)
       at com.onesignal.ActivityLifecycleHandler.onActivityResumed(ActivityLifecycleHandler.java:127)
       at com.onesignal.ActivityLifecycleListener.onActivityResumed(ActivityLifecycleListener.java:78)
       at android.app.Application.dispatchActivityResumed(Application.java:239)
       at android.app.Activity.onResume(Activity.java:1362)
       at com.unity3d.player.UnityPlayerActivity.onResume()
       at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1434)
       at android.app.Activity.performResume(Activity.java:8762)
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3829)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3869)
       at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6852)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:504)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

```

**Comment by Jeasmine** [comment]
Thanks for submitting the error, we had some Location fixes lately like https://github.com/OneSignal/OneSignal-Android-SDK/commit/d25a6727bb20ddfbf55a7231278f1dc6406fe97e this will be included on the next release and is likely to fix your problem

**Comment by andpanda** [comment]
> Thanks for submitting the error, we had some Location fixes lately like [d25a672](https://github.com/OneSignal/OneSignal-Android-SDK/commit/d25a6727bb20ddfbf55a7231278f1dc6406fe97e) this will be included on the next release and is likely to fix your problem

Thanks Jeasmine, this is very good news.

**Comment by Jeasmine** [comment]
Close due to a new crash related, follow new thread under https://github.com/OneSignal/OneSignal-Android-SDK/issues/1170
